templates/todo/projects/python-mongo/.repo/terraform/infra/database.tf

```diff
@@ -0,0 +1,71 @@
+# ------------------------------------------------------------------------------------------------------

```

Comment:
Let's put these in a "modules" subfolder.  So, "main" and "resources" should be the only files in the root of infra.

Reply:
We decided to keep flat for now because with TF if you put in any subfolder then it requires you create it as a special TF module which requires more files and more work.